File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        store = set()
        for i in nums:
            store.add(i)
        ans = 0
        for num in store:
            if num-1 in store:
                continue
            else:
                next_el = num+1
                count = 1
                while(True):
                    if next_el in store:
                        count+=1
                        next_el+=1
                    else:
                        ans = max(ans, count)
                        break
        return ans

File: Data Structures & Algorithms/missing-number/submission-0.java
class Solution {
    public int missingNumber(int[] nums) {
        int xor = 0;

        for (int i : nums) {
            xor ^= i;
        }

        int n = nums.length;

        for (int i = 1; i <= n; i++) {
            xor ^= i;
        }

        return xor;
    }
}
